src/app/layouts/stepper/stepper-overview-sample/stepper-overview-sample.component.html

```diff
@@ -0,0 +1,262 @@
+                        <span class="card-currency">{{card.price}}</span>
+                        <span class="card-title">{{card.offer}}</span>
+                        <p>{{card.type}}</p>
+                        <p>{{card.description}}</p>
+                    </igx-card-content>
+                </igx-card>
+            </div>
+        </div>
+    </igx-step>
+
+    <igx-step #step2 [isValid]="businessInformationForm.form.valid" [completed]="businessInformationForm.form.valid">
+        <span igxStepTitle>Business information</span>
+        <div igxStepContent [tabIndex]="-1">
+            <div class="sample-row">
+                <form class="flex-form" #businessInformationForm="ngForm">
+                    <igx-input-group type="box">
+                        <input igxInput name="name" id="name" type="text" [(ngModel)]="businessInformation.name"
+                            required />
+                        <label igxLabel for="name">Legal Business Name</label>
+                    </igx-input-group>
+                    <igx-input-group type="box">
+                        <input igxInput name="physicalAddress" id="physicalAddress" type="text"
+                            [(ngModel)]="businessInformation.physicalAddress" required />
+                        <label igxLabel for="physicalAddress">Business Physical Address</label>
+                    </igx-input-group>
+                    <div class="form-row">
+                        <igx-input-group type="box">
+                            <input igxInput name="city" id="city" type="text" [(ngModel)]="businessInformation.city"
+                                required />
+                            <label igxLabel for="city">City</label>
+                        </igx-input-group>
+                        <igx-select type="box" name="state" [(ngModel)]="businessInformation.state" required>
+                            <igx-select-item *ngFor="let state of states" [value]="state">
+                                {{state}}
+                            </igx-select-item>
+                            <label igxLabel>State</label>
+                        </igx-select>
+                        <igx-input-group type="box">
+                            <input igxInput name="zip" id="zip" type="text" pattern="[0-9]{5}" [igxMask]="'00000'"
+                                [placeholder]="'#####'" [(ngModel)]="businessInformation.zip" required />
+                            <label igxLabel for="zip">ZIP Code</label>
+                        </igx-input-group>
+                    </div>
+                    <igx-checkbox name="differentAddress" [(ngModel)]="businessInformation.differentAddress">
+                        The mailing address is different than the business physical address
+                    </igx-checkbox>
+                    <igx-input-group type="box">
+                        <input igxInput name="taxIdNumber" id="taxIdNumber" type="text" pattern="[9]{1}[0-9]{8}"
```

Comment:
Might be a good idea to add a little hint that the expected number is supposed to start with a `9` because some users might not know that and will have to look it up online, which can be a bit tedious.